### PR TITLE
Have the SalsaConnection post return the response

### PR DESCRIPTION
**Pull request by toppa**
Needed for changes in progress with Indigo event notifications

### CODE REVIEW

File: lib/wired_for_change/salsa_connection.rb [hunk 1]
```diff
@@ -35,6 +35,7 @@ def _post(object, assertive=false)
     if assertive && ! post_succeeded?
       raise PostFailure, "No success entity in response to post"
     end
+    post_response
```
Comment (wimdenbraven) [inline lib/wired_for_change/salsa_connection.rb]: what do you want to return if code != 200 (line 32)?

**Comment by wimdenbraven** [comment]
this is a public gem and _post is not a private method, therefore I'd except a change in the minor version number

File: lib/wired_for_change/salsa_connection.rb (hunk 1 above)
Comment (toppa) [inline lib/wired_for_change/salsa_connection.rb]: Hmmm.. thanks for noticing that. For my purposes I'd like to return the `post_response` there as well. I'll add that.

**Comment by toppa** [comment]
Good catch - I already bumped the version (I squashed it already, so it wasn't obvious)

**Comment by cluesque** [comment]
👍

**Pull request merged**